src/edge/simulator.ts

```diff
@@ -91,85 +56,90 @@ export class Simulator {
         return null;
     }
 
-    private static async getLatestSimulatorVersion(): Promise<string | undefined> {
+    private static async checkCmdExist(cmd: string): Promise<boolean> {
```

Comment:
add a string check here so if string is undefined/null just return false without exception.

Reply:
OK